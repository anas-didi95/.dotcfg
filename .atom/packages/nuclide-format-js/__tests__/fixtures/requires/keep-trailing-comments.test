const B = require('B'); // fancy B

import type {Z} from 'Z'; // something else

const A = require('A'); // huge A

const x: Z = A + B;
