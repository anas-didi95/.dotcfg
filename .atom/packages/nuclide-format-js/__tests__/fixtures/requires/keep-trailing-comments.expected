import type {Z} from 'Z'; // something else

const A = require('A'); // huge A
const B = require('B'); // fancy B

const x: Z = A + B;
